website/common/script/ops/sell.js

```diff
@@ -9,6 +9,7 @@ import {
   BadRequest,
 } from '../libs/errors';
 
+// @TODO: 'special' type throws NotAuthorized error
```

Comment:
isn't this ok since ACCEPTEDTYPES doesn't list it?

Reply:
I was trying to sell a Saddle on local that appeared in the `special` tab and it threw this error. 

Maybe this could be re-worded, or moved into the market view and state that `special` types currently aren't in the `ACCEPTEDTYPES` const and will break.